### ISSUE TITLE
Kubernetes Go modules are not tagged for v1.23.0-beta.0

### ISSUE THREAD

**Issue by smira** [kubernetes/kubernetes#106529]
### What happened?

New releases is not tagged.

### What did you expect to happen?

Tag should be there.

### How can we reproduce it (as minimally and precisely as possible)?

https://pkg.go.dev/k8s.io/api?tab=versions

```
$ go get k8s.io/api@v1.23.0-beta.0
go: k8s.io/api@v0.23.0-beta.0: reading k8s.io/api/go.mod at revision v0.23.0-beta.0: unknown revision v0.23.0-beta.0
```

### Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

### Cloud provider

<details>

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by neolit123** [comment]
looks like this is being discussed on Slack for potential fixes:
https://kubernetes.slack.com/archives/CJH2GBF7Y/p1637586055203100

**Comment by justaugustus** [comment]
Establishing some ownership here:

1.23 Branch Managers:
/assign @Verolop @palnabarun 

/area release-eng
/triage accepted
/priority critical-urgent
cc: @kubernetes/release-engineering @kubernetes/publishing-bot-reviewers 
xref: https://github.com/kubernetes/sig-release/pull/1504#pullrequestreview-810342941

**Comment by liggitt** [comment]
weird... the 1.24... tags showed up, but not the 1.23 beta or rc tags:

https://github.com/kubernetes/api/tags

**Comment by liggitt** [comment]
once the 1.23 branch config is added to the publishing rules, I'd expect the 1.23 rc tags to sync, but I can't explain why the 1.23 beta tags didn't

**Comment by liggitt** [comment]
this is resolved now, starting with v0.23.0-rc.X tags
